File: Two to One/two-to-one.py
def longest(s1, s2):
    """
    :type s1: str
    :type s2: str
    :rtype: str
    """
    return "".join(sorted(list(set(s1 + s2))))


print(longest("aretheyhere", "yestheyarehere"))  # "aehrsty"
print(longest("loopingisfunbutdangerous", "lessdangerousthancoding"))  # "abcdefghilnoprstu"

File: Two to One/two-to-one.js
/**
 * Find a new sorted string, the longest possible, containing distinct letters
 *
 * @return {string}
 * @param a {string}
 * @param b {string}
 */
const longest = (a, b) => {
  return [...new Set(a + b)].sort().join('');
};

console.log(longest('aretheyhere', 'yestheyarehere')); // 'aehrsty'
console.log(longest('loopingisfunbutdangerous', 'lessdangerousthancoding')); // 'abcdefghilnoprstu'